src/Behat/Testwork/Call/RuntimeCallee.php

```diff
@@ -116,4 +116,25 @@ public function isAnInstanceMethod()
         return $this->reflection instanceof ReflectionMethod
             && !$this->reflection->isStatic();
     }
+
+    public function throwIfInstanceMethod(callable|array $callable, string $hookType): void
```

Comment:
Should perhaps have the full $callable type definition and be a protected, rather than public, method?

```suggestion
    /**
     * @param callable|array{class-string, string} $callable
     */
    protected function throwIfInstanceMethod(callable|array $callable, string $hookType): void
```

Reply:
Ah, yes of course, it needs to be protected, not public 🤦  And adding the full type definition makes sense, updated